cmake/Sanitizers.cmake

```diff
@@ -10,6 +10,11 @@ function(
   if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES ".*Clang")
     set(SANITIZERS "")
 
+    if(CMAKE_BUILD_TYPE STREQUAL "Release")
```

Comment:
However here is: if RELEASE.

So `RelWithDebInfo` and `MinSizeRel` will be uncovered.